client/src/app/services/auth.service.ts

```diff
@@ -73,6 +73,11 @@ export class AuthService {
     return this.http.put(this.domain + 'authentication/authorizationChange', users).map(res => res.json());
   }
 
+  sendInvitation(invitation) {
+    this.createAuthenticationHeaders(); 
+    return this.http.post(this.domain + 'authentication/sendInvitation', invitation, this.options).map(res => res.json());
+  } 
+
```

Comment:
this should not in auth service, as the service grow larger we need a section to better plan our API